### DISCUSSION TITLE
I want to ask how the network map the Descriptor into the output energy,

### DISCUSSION THREAD

**Discussion by DM0815** [deepmodeling/deepmd-kit#2127] [Q&A]
### Summary

Because I want to figure out the deepMD network how to work and output the energy. In other words, I know how to get the env matrix and know the equation about rotation,translation, and permutation. but after these steps, we need  input the descriptor D to network(fitting netwotk N) and get the output 'energy'. But I don't konw the processing in fitting network . I also saw the Dr.zhang's papers about DeepMD. However, the paper didn't illustrate the fitting processing in detail. Can you help me and explain it. Thank you very much. 

### DeePMD-kit Version

dp --2.1.5

### TensorFlow Version

tf --2.11.0

### Python Version, CUDA Version, GCC Version, LAMMPS Version, etc

py3.6

### Details

Because I want to figure out the deepMD network how to work and output the energy. In other words, I know how to get the env matrix and know the equation about rotation,translation, and permutation. but after these steps, we need  input the descriptor D to network(fitting netwotk N) and get the output 'energy'. But I don't konw the processing in fitting network . I also saw the Dr.zhang's papers about DeepMD. However, the paper didn't illustrate the fitting processing in detail. Can you help me and explain it. Thank you very much.

**Comment by njzjz** [comment]
The explanation is available in the documentation.

https://docs.deepmodeling.com/projects/deepmd/en/v2.1.5/api_py/deepmd.fit.html#deepmd.fit.ener.EnerFitting
